statsmodels/discrete/discrete_model.py

```diff
@@ -1678,6 +1730,89 @@ def hessian(self, params):
 
         return hess_arr
 
+    def hessian_factor(self, params):
+        """
+        Generalized Poisson model Hessian matrix of the loglikelihood
```

Comment:
I'm not clear on how this is different from just `hessian`.  My understanding of `hessian_factor` is it gives the `X` such that `exog.dot(X).dot(exog)` gives `hessian` or something like that.

Reply:
hessian factor is 1-D, and can also be multiplied with other exog than the ones in the model, which is what I use in the score_test and conditional moment tests